Pod/Classes/WPMediaCaptureCollectionViewCell.h

```diff
@@ -1,6 +1,6 @@
 @import UIKit;
 
-@interface WPMediaCaptureCollectionViewCell : UICollectionViewCell
+@interface WPMediaCaptureCollectionViewCell : UICollectionReusableView
```

Comment:
Should this be named something other than `Cell`, now it's no longer a cell subclass? `WPMediaCaptureCollectionViewHeader` maybe? ``WPMediaCaptureCollectionViewHeaderView`? ``WPMediaCaptureCollectionViewHeaderFooterView`? :)